docs/user-manual/gaussian-splatting/creating/sog-format.md

```diff
@@ -0,0 +1,297 @@
+| `shN_centroids.webp` | SH palette coefficients (optional)  | RGBA             |
+
+> **Image formats.**
+> By default, images SHOULD be **lossless WebP** to preserve quantized values exactly.
+> Each property in `meta.json` names its file, so other 8-bit RGBA-capable formats MAY be used.
+> Do not use lossy encodings for these assets as lossy compression will corrupt values and can produce visible/structural artifacts.
+
+### 1.1 Image dimensions & indexing
+
+All per-Gaussian properties are co-located: the same pixel (x,y) across all property images (except shN_centroids) belongs to the same Gaussian.
+
+* Pixels are laid out **row-major**, origin at the **top-left**.
+* For image width `W` and height `H`, the number of addressable gaussians is `W*H`.
+* `meta.count` MUST be `<= W*H`. Any trailing pixels are ignored.
+
+**Indexing math (zero-based):**
+
+* From index to pixel:
+  `x = i % W`, `y = floor(i / W)`
+* From pixel to index:
+  `i = x + y * W`
+
+### 1.2 Coordinate system
+
+Right-handed:
+
+* **x:** right
+* **y:** up
+* **z:** back (i.e., −z is “forward” in camera-looking-down −z conventions)
+
+### 1.3 Bundled variant
+
+A bundled SOG is a ZIP of the files above. Readers SHOULD accept either layout:
+
+* **Multi-file directory** (recommended during authoring)
+* **Single archive** (e.g., `scene.sog`) containing the same files at the archive root
+
+Readers MUST unzip and then resolve files using `meta.json` exactly as for the multi-file version.
+
+---
+
+## 2) `meta.json`
+
+```ts
+interface Meta {
+  version: 2;              // File format version (integer)
+  count: number;           // Number of gaussians (<= W*H of the images)
+  antialias: boolean;      // True iff scene was trained with anti-aliasing
+
+  means: {
+    // Ranges for decoding *log-transformed* positions (see §3.1).
+    mins: [number, number, number];   // min of nx,ny,nz (log-domain)
+    maxs: [number, number, number];   // max of nx,ny,nz (log-domain)
+    files: ["means_l.webp", "means_u.webp"];
+  };
+
+  scales: {
+    codebook: number[];    // 256 floats; see §3.3
+    files: ["scales.webp"];
+  };
+
+  quats: {
+    files: ["quats.webp"]; // §3.2
+  };
+
+  sh0: {
+    codebook: number[];    // 256 floats; maps quantized DC to linear color (§3.4)
+    files: ["sh0.webp"];
+  };
+
+  // Present only if higher-order SH exist:
+  shN?: {
+    count: number;         // Palette size (up to 65536)
+    bands: number;         // Number of SH bands (1..3). DC (=band 1) lives in sh0.
+    codebook: number[];    // 256 floats; shared for all AC coefficients (§3.5)
+    files: [
+      "shN_labels.webp",   // Per-gaussian palette index (0..count-1)
+      "shN_centroids.webp" // Palette of AC coefficients as pixels (§3.5)
+    ];
+  };
+}
+```
+
+> **Notes**
+>
+> * All codebooks contain linear-space values, not sRGB.
+> * Image data MUST be treated as raw 8-bit integers (no gamma conversion).
+> * Unless otherwise stated, channels not mentioned are ignored.
```

Comment:
```suggestion
:::note
* All codebooks contain linear-space values, not sRGB.
* Image data **must** be treated as raw 8-bit integers (no gamma conversion).
* Unless otherwise stated, channels not mentioned are ignored.
:::
```